web/src/components/Upstream/UpstreamForm.tsx

```diff
@@ -728,8 +727,14 @@ const UpstreamForm: React.FC<Props> = forwardRef(
                     <Form.Item
                       label={formatMessage({ id: 'page.upstream.step.pass-host.upstream_host' })}
                       name="upstream_host"
+                      rules={[
+                        {
+                          required: true,
+                          message: "",
```

Comment:
why the message is an empty string? 🤔

Reply:
because when users leave this field empty, the input box will have a red border to alert them 🤔

Reply:
it seems that when the required field is empty,  the required message will appear behind the input box, so if we leave the message an empty string, there will be an empty line appear. anyway,  since the `host rewrite` become a disable option, this input box will never show in the page.

Reply:
ya